segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/mocks/InMemorySnapshotInfoStore.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.pravega.segmentstore.storage.mocks;
+
+import io.pravega.segmentstore.storage.chunklayer.SnapshotInfo;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class InMemorySnapshotInfoStore {
```

Comment:
Javadoc please. And explain this is for testing purposes only.

Also, would this belong in the `test` project? Is it used anywhere outside of tests?

Reply:
Moved to tests.